sdk-commons/src/main/java/org/openmrs/maven/plugins/model/Artifact.java

```diff
@@ -177,7 +180,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return groupId + ':' + artifactId + ':' + version;
+        StringBuilder sb = new StringBuilder();
```

Comment:
Don't use a `StringBuilder` the string concatenation here is actually the right concatenation technique to use.